### PR TITLE
modulebuilder

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

File: htdocs/modulebuilder/template/admin/about.php [hunk 1]
```diff
@@ -23,18 +23,7 @@
  */
 
 // Load Dolibarr environment
-$res = 0;
```
Comment (eldy) [inline htdocs/modulebuilder/template/admin/about.php:26]: Please separate PR that change the code syntaxt from PR that has effect on architecture change.

Comment (frederic34) [reply htdocs/modulebuilder/template/admin/about.php:26]: https://github.com/Dolibarr/dolibarr/pull/15915

**Comment by eldy** [comment]
@frederic34 What means the PR with status "Draft". Does it means, i have to wait, YOU set the status of PR to another status before i merge it ? Or do you wait something from me ?

**Comment by frederic34** [comment]
@eldy I set to draft if I think it's not ready to merge, I will set back to merge if ready

File: htdocs/modulebuilder/template/config.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+// Defines
+$defines = (isset($defines) && is_array($defines)) ? $defines : [];
+foreach ($defines as $define) {
+	if (! defined($define)) {
+		define($define, 1);
+	}
+}
+// Load Dolibarr environment
+$res = 0;
+$filename = isset($_SERVER['SCRIPT_FILENAME']) ? explode('/', parse_url($_SERVER['SCRIPT_FILENAME'])['path']) : [];
```
Comment (eldy) [inline htdocs/modulebuilder/template/config.php:28]: $_SERVER['SCRIPT_FILENAME'] is a path not an url. parse_url will not be able to manage correctly symbolic links.
What if SCRIPT_FILENAME is empty ?

**Pull request closed without merging**